### ISSUE TITLE
Redundant dependency

### ISSUE THREAD

**Issue by Sam-Kruglov** [smcvb/gamerental#1]
This one is already inside the extension

https://github.com/smcvb/gamerental/blob/912cf109db41f026c9d550ffc914e65f06100013/pom.xml#L52

P.S. very cool :)

**Comment by smcvb** [comment]
You're completely right! I'll clear it out from the sample. Thanks for pointing this out @Sam-Kruglov!

**Comment by smcvb** [comment]
Resolved the problem, so closing this issue. Thanks again for setting it up @Sam-Kruglov!
